srcpkgs/odin/template

```diff
@@ -20,7 +20,9 @@ make_use_env=yes
 do_install() {
 	vmkdir opt/odin
 	vmkdir usr/bin
-	vcopy "*" opt/odin
+	vcopy "shared" opt/odin
```

Comment:
Is this really needed? It just includes an empty `.gitkeep`.

Reply:
I only just started messing around with the language, but their github readme explicitly states that core and shared library folders need to stay relative to the executable.

Reply:
This looks like to be a placeholder where `.so` (or `.odin`?) files can be dropped for importing from within odin, so maybe just keep this.
(I had a concern that odin maybe uses this directory for temporary storing shared objects before linking the final binary. This would be problematic with a root-owned `/opt`.)